tools/Cron_jobs/mailwatch_geoip_update.php

```diff
@@ -170,7 +170,7 @@
 
         // Apply MailWatch rights on files from the last run
         $mwUID =  exec('cat /etc/sudoers.d/mailwatch | grep "User_Alias MAILSCANNER" | sed "s/.*= \(.*\).*/\1/"', $retval_cat);
-        if ($retval_cat > 0) {
```

Comment:
$retval_cat is an INT, why do you check as an array?
http://php.net/manual/en/function.exec.php